src/utils/rootkey.ts

```diff
@@ -87,7 +87,6 @@ export function mapApiResponseToObjects(
           return {
             signingKey: obj.signing_key,
             type: Network.NEAR,
-            chainId: obj.wallet.chainId ?? 1,
             createdAt: obj.created_at,
           } as NearRootKey;
```

Comment:
How did it compile? `NearRootKey` does not have `chainId`

Reply:
Not sure..

Reply:
Gets compiled both ways, explanation:
`When returning the object { signingKey, type, createdAt, bubble: "abc" } with the as NearRootKey assertion, TypeScript does not check if bubble is a valid property of NearRootKey. It trusts that the object matches the NearRootKey interface because of the assertion.`
<img width="1078" alt="Screenshot 2024-07-02 at 09 26 55" src="https://github.com/calimero-network/admin-dashboard/assets/93442516/50d59c16-8bee-4100-b218-0c901e04c25b">

Reply:
let's then use a format that will warn about it

Reply:
updated